proto/osmosis/incentives/gauge.proto

```diff
@@ -9,33 +9,37 @@ import "osmosis/lockup/lock.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v7/x/incentives/types";
 
+// Gauge is a struct that works as a unit for each distribution
+// information defined by x/incentives module.
 message Gauge {
-  // unique ID of a Gauge
+  // ID is the unique id of the gauge.
+  // The ID of the gauge is decided upon gauge creation, incrementing by 1 for
+  // every gauge.
   uint64 id = 1;
-  // flag to show if it's perpetual or multi-epoch
-  // distribution incentives by third party
+  // Is Perpetual defines whether the gauge is permanent or multi-epoch
+  // distribution.
   bool is_perpetual = 2;
```

Comment:
Lets in-line define what this means, and the context of this on other parameters. (Or we can define the two gauge types on the main comment description for the `Gauge` struct.

Reply:
updated comments on this, lmk what you think!